### PR TITLE
work on handle all the projects and modifying all files of a  du

### PR DESCRIPTION
**Pull request by ivanvolkoff**
This PR fixes handling/modification for multiple projects in a delivery unit.

### CODE REVIEW

File: ide-migration/server/migration/api/migration-service.js [hunk 1]
```diff
@@ -365,6 +354,48 @@ class MigrationService {
         hdiFile.setText(hdiJson);
     }
 
+    addFilesWithoutGenerated(userData, workspace, locals) {
+        for (const local of locals) {
```
Comment (vmutafov) [inline ide-migration/server/migration/api/migration-service.js:358]: Could you rename `locals` to `localFiles` and `local` to `localFile`. Same goes for other loops

File: ide-migration/server/migration/api/migration-service.js [hunk 2]
```diff
@@ -365,6 +354,48 @@ class MigrationService {
...
+                git.initRepository('migration', '', workspace, projectName, projectName, "Migration initial commit");
+            }
+        }
+    }
+
+    addGeneratedFiles(userData, deliveryUnit, workspace, locals) {
+        for (const local of locals) {
+            const projectName = local.projectName;
+            const generated = deliveryUnit['deployableArtifactsResult']['generated'].filter(x => x.projectName === projectName);
+            for (const gen of generated) {
```
Comment (vmutafov) [inline ide-migration/server/migration/api/migration-service.js:382]: Could you rename `generated` to `generatedFiles` and `gen` to `generatedFile`?

**Review by vmutafov — CHANGES_REQUESTED** [review]

File: ide-migration/server/migration/api/migration-service.js (hunk 1 above)
Comment (ivanvolkoff) [reply ide-migration/server/migration/api/migration-service.js:358]: +1

File: ide-migration/server/migration/api/migration-service.js (hunk 2 above)
Comment (ivanvolkoff) [reply ide-migration/server/migration/api/migration-service.js:382]: +1

**Review by tdermendjiev — APPROVED** [review]

**Review by vmutafov — APPROVED** [review]

**Pull request merged**